components/compositing/compositor_thread.rs

```diff
@@ -28,6 +28,42 @@ pub trait EventLoopWaker : 'static + Send {
     fn wake(&self);
 }
 
+/// Sends messages to the embedder.
+pub struct EmbedderProxy {
```

Comment:
Re this, I was looking into it and realized this channel is more used between the Constellation and the embedder(or rather Servo). Also, the current name is quite consistent with the `CompositorProxy` in the same package, which is used by the Constellation as well. Perhaps we should keep this as it is?